### PR TITLE
Made base-uri confable via default.js and live.js

### PR DESCRIPTION
**Pull request by m0ru**
It's not drop-in confable like the skin-folder, but I wanted to avoid introducing another round-trip to the page load and keep the application statically servable.

Note: can't properly test this, as my setup's fubar atm. The work for this PR was an attempt to circumvent this by using an alternative owner-server like integration-testing.

### CODE REVIEW

**Comment by m0ru** [comment]
Not mergeable yet. @peacememories and I ran into some probs while testing it on his machine. We'll look at it again tomorrow.

File: webofneeds/won-owner-webapp/src/main/webapp/app/components/connection-message.js [hunk 1]
```diff
@@ -154,14 +156,14 @@ function genComponentConf() {
...
-                href="/owner/rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}&deep=true">
+                href="{{self.ownerBaseUrl}}rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}&deep=true">
                     <svg class="rdflink__small clickable">
                             <use xlink:href="#rdf_logo_2" href="#rdf_logo_2"></use>
                     </svg>
             </a>
             <a ng-show="self.shouldShowRdf && !self.message.get('outgoingMessage')"
                 target="_blank"
-                href="/owner/rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}">
+                href="{{self.ownerBaseUrl}}rest/linked-data/?requester={{self.encodeParam(self.ownNeed.get('uri'))}}&uri={{self.encodeParam(self.message.get('uri'))}}">
```
Comment (quasarchimaere) [inline webofneeds/won-owner-webapp/src/main/webapp/app/components/connection-message.js]: please change the ownerBaseUrl so it does NOT expect a trailing slash at the end, that way the links wont break even if the conf param has a trailing slash at the end

**Comment by peacememories** [comment]
I think the problem we ran into is because our input should look like this:
`import ownerBaseUrl from 'config'`
instead of
`import {ownerBaseUrl} from 'config'`
because we are not deconstructing an object but importing a certain name.

File: webofneeds/won-owner-webapp/src/main/webapp/app/components/connection-message.js (hunk 1 above)
Comment (m0ru) [reply webofneeds/won-owner-webapp/src/main/webapp/app/components/connection-message.js]: mhm, good point
after trying a few variations, i introduced `urljoin` that handles path-concatenation correctly.

**Comment by m0ru** [comment]
@quasarchimaere  mhm, good point
after trying a few variations, i introduced urljoin that handles path-concatenation correctly.
everything should work now as intended on the client side. still needs some changes to spring's security settings to actually leverage this -- for now it just works as before but with less string-literal duplication.

**Pull request merged**